docs/source/user-guide/integrations/spark_databricks.rst

```diff
@@ -0,0 +1,170 @@
+
+    # The following two lines are estimators. They return functions that we will later apply to transform the dataset.
+    stringIndexer = StringIndexer(inputCols=categoricalCols, outputCols=[x + "Index" for x in categoricalCols])
+    encoder = OneHotEncoder(inputCols=stringIndexer.getOutputCols(), outputCols=[x + "OHE" for x in categoricalCols])
+
+    # The label column ("income") is also a string value - it has two possible values, "<=50K" and ">50K".
+    # Convert it to a numeric value using StringIndexer.
+    labelToIndex = StringIndexer(inputCol="income", outputCol="label")
+
+    stringIndexerModel = stringIndexer.fit(trainDF)
+
+    # This includes both the numeric columns and the one-hot encoded binary vector columns in our dataset.
+    numericCols = ["age", "fnlwgt", "education_num", "capital_gain", "capital_loss", "hours_per_week"]
+    assemblerInputs = [c + "OHE" for c in categoricalCols] + numericCols
+    vecAssembler = VectorAssembler(inputCols=assemblerInputs, outputCol="features")
+
+Training the Model
+~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    from pyspark.ml.classification import LogisticRegression
+    from pyspark.ml import Pipeline
+
+    lr = LogisticRegression(featuresCol="features", labelCol="label")
+
+    # Define the pipeline based on the stages created in previous steps.
+    pipeline = Pipeline(stages=[stringIndexer, encoder, labelToIndex, vecAssembler, lr])
+
+    # Define the pipeline model.
+    pipelineModel = pipeline.fit(trainDF)
+
+Writing a Model Wrapper
+~~~~~~~~~~~~~~~~~~~~~~~
+We will write a wrapper to our model, that will implement the required API for deepchecks according the the
+:doc: `</user-guide/tabular/supported_models>` guide.
+
+.. code-block:: python
+
+    import numpy as np
+    import pyspark
+    from pyspark.ml.feature import IndexToString
+
+    class PySparkModelWrapper:
+        def __init__(self, model: pyspark.ml.pipeline.PipelineModel, label_map):
+            self.model = model
+            self.idx_to_string = IndexToString(inputCol="prediction", outputCol="predictedValue")
+            self.idx_to_string.setLabels(label_map)
+
+        def predict(self, X: np.ndarray) -> np.ndarray:
+            df=spark.createDataFrame(X)
+            preds = self.idx_to_string.transform(self.model.transform(df).select('prediction')).select('predictedValue').collect()
+            return np.array(preds).reshape(-1)
+
+        def predict_proba(self, X: np.ndarray) -> np.ndarray:
+            df=spark.createDataFrame(X)
+            preds = self.model.transform(df).select('prediction').collect()
+            return np.array(preds).reshape(-1, 2)
+
+        @property
+        def feature_importances_(self):
+            return np.array([1/14] * 14)
+
+.. note::
+    The wrapper here considers that all features are equally important. This is not a valid assumption for
+    real models, but is done here for simplicity.
+
+Running the Full Suite
```

Comment:
Lets replace this with a model evaluation suite and (/or) a train-test validation suite
(to "publicize" less the full suite)